ipmininet/router/config/bgp.py

```diff
@@ -118,7 +120,7 @@ class AddressFamily(object):
     def __init__(self, af_name, redistribute=(), networks=(),
                  *args, **kwargs):
         self.name = af_name
-        self.networks = map(ip_network, map(unicode, networks))
+        self.networks = list(map(ip_network, map(str, networks)))
```

Comment:
I'd like to avoid the double list expansion that happens in py2 were possible.
List comprehensions would fit the bill here:

```python
self.networks = [ip_network(str(n)) for n in networks]
```